.github/workflows/post-commit.yml

```diff
@@ -238,3 +238,61 @@ jobs:
            name: coverage
            path: target/coverage
          continue-on-error: true
+  kubernetes:
+    name: kubernetes
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: maven-repo-${{ hashFiles('**/pom.xml') }}
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.pnpm-store
+            **/node_modules
+          key: ${{ runner.os }}-pnpm-${{ hashFiles('**/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-pnpm-
+      - name: checkout to /mnt/ozone
+        run: |
+          sudo chmod 777 /mnt
+          git clone https://github.com/${GITHUB_REPOSITORY}.git /mnt/ozone
+          cd /mnt/ozone
+          git fetch origin "${GITHUB_REF}"
+          git checkout FETCH_HEAD
```

Comment:
Can you please update this with the fix from fd47f915c65512de6618811e6c247871066a8351?

Reply:
fair point. Replaced with the new version. (One more reason to move checkout logic to a separated script like https://github.com/elek/hadoop-ozone/blob/smart-checkout/.github/checkout.sh)